src/cli/cli_dataset.cpp

```diff
@@ -94,6 +95,13 @@ ThreadError Dataset::Print(otOperationalDataset &aDataset)
         sServer->OutputFormat("Channel: %d\r\n", aDataset.mChannel);
     }
 
+    if (aDataset.mIsChannelMaskPage0Set)
+    {
+        sServer->OutputFormat("Channel Mask Page 0: ");
+        OutputBytes(reinterpret_cast<uint8_t *>(&aDataset.mChannelMaskPage0), sizeof(aDataset.mChannelMaskPage0));
```

Comment:
Probably should just use `OutputFormat()` now that `mChannelMaskPage0` is just a `uint32_t`.